module-2/lab-hypothesis-testing/your-code/challenge-2.ipynb

```diff
@@ -246,15 +246,39 @@
   },
```

Comment:
-"First let's obtain the unique values of the pokemon types. These values should be extracted from Type 1 and Type 2 aggregated. Assign the unique values to a variable called unique_types." Otra forma sin usar set podría ser: 
```Python
types = pokemon['Type 1'].append(pokemon['Type 2']).unique()
unique_types = types.tolist()
len(unique_types)
```
- Otra forma más sencilla de calcular el pokemon_totals. Ya tienes calculado unique_types no hace falta que hagas el doble bucle creo yo. 
```Python
pokemon_totals = []
for ptype in unique_types:
    if isinstance(ptype, str):
        pokemon_totals.append(pokemon.loc[(pokemon['Type 1']==ptype) | (pokemon['Type 2']==ptype)]['Total'])
len(pokemon_totals)
```
- En general, cuando corras un contraste de hipótesis, es conveniente que saques todos los resultados no solo el p-value sino también el estadístico de contraste, en este caso, la F de Fisher.